src/service/resources-impl.js

```diff
@@ -1495,8 +1476,7 @@ export class Resource {
     /** @const {!AmpElement|undefined|null} */
     this.owner_ = undefined;
 
-    /** @const {number} */
-    this.priority_ = getElementPriority(element.tagName);
+    this.priority_ = element.getElementPriority();
```

Comment:
None. You can just get rid of this instant variable and instead call the method where you need the value.